### ISSUE TITLE
Highlight max size is too small

### ISSUE THREAD

**Issue by mayfield** [shutter-project/shutter#719]
#### I'm on a modern high res display and the max size is more like a pen than a highlighter.  I don't see any reason why this tool should have any max size to be honest.  I would set it to 100+ pixels in some cases.

**Comment by crojack** [comment]
> #### I would like to use highlight with a significantly large brush size than 20px.
> #### I'm on a modern high res display and the max size is more like a pen than a highlighter. I don't see any reason why this tool should have any max size to be honest. I would set it to 100+ pixels in some cases.

Go to /usr/share/shutter/resources/modules/Shutter/Draw/DrawingTool.pm and search for SpinButton and specifically this line: _$self->{_line_spin_w} = Gtk3::SpinButton->new_with_range(1.0, 20.0, 0.5);_
and change the value from 20.0 to 300.0 .

![Screenshot from 2024-12-17 22-00-34](https://github.com/user-attachments/assets/64eaa9dc-d142-41a9-8ded-54736f067d88)

**Comment by Photon89** [comment]
In case you want to continue using your already installed Shutter version, you can edit `DrawingTool.pm` directly in `/usr/share/shutter/resources/modules/Shutter/Draw/`.

**Comment by crojack** [comment]
> In case you want to continue using your already installed Shutter version, you can edit `DrawingTool.pm` directly in `/usr/share/shutter/resources/modules/Shutter/Draw/`.

Of course, thanks :). I've edited my post above. Too tired to think, lol. You can close this issue now, right?

**Comment by Photon89** [comment]
Well, maybe we should change this value in master to allow a thicker highlighter for everybody?

**Comment by crojack** [comment]
> Well, maybe we should change this value in master to allow a thicker highlighter for everybody?

Of course. He is right, on big monitors that run 4k+ resolutions highlighter lines look like pen lines, not like highlighter lines. 

I have a question. It seems like you are the only active developer here, right? I've seen a few more issues that should be closed.

**Comment by Photon89** [comment]
I guess, I am the most active team member for some periods of time, but the least experienced one, so I always ask @DarthGandalf for code reviews before I merge anything. :smile:
